ClosedXML/Excel/CalcEngine/Functions/Statistical.cs

```diff
@@ -207,6 +207,11 @@ private static object CountIfs(List<Expression> p)
             return count;
         }
 
+        private static object Fisher(List<Expression> p)
+        {
+            return GetTally(p, true).Fisher();
```

Comment:
Since this function takes only 1 parameter, I don't think it's necessary to use the `Tally` class, right?

Reply:
In this case, we are using the Tally class, which has a some useful logic, so I used that. In this case, it filters out non-numeric parameters. I think we should use it.

Another thing is that this class may have a somewhat inappropriate name and violates the principle of single responsibility. I think it make sense in future move logic to statistical function in Statistical class and logic for check parameters leave on this class and after this rename it. It make sense?

Reply:
Initially I don't correct understand you and I removed my old message.

Reply:
Now I updated this code and I don't use now Tally class.

This is working fine but I don't sure about this:
`double.TryParse(p?[0].Evaluate().ToString(), out double value)`
Is this a good solution for checking if a value is a double? I don't sure. If not please tell me about good way and I will update this check. In this moment I don't good know this codebase.

Thanks.